packages/docusaurus-theme-openapi-docs/src/theme/ParamsDetails/index.tsx

```diff
@@ -72,14 +72,7 @@ const ParamsDetails: React.FC<Props> = (props) => {
   return (
     <BrowserOnly fallback={<SkeletonLoader size="sm" />}>
       {() => {
-        const LazyComponent = React.lazy(() =>
-          Promise.resolve({ default: ParamsDetailsComponent })
-        );
-        return (
-          <Suspense fallback={null}>
-            <LazyComponent {...props} />
-          </Suspense>
-        );
+        return <ParamsDetailsComponent {...props} />;
```

Comment:
Removed use of `Suspense` for handling lazy loading of component...now relying strictly on `BrowserOnly`.